lib/ndr_import/universal_importer_helper.rb

```diff
@@ -35,7 +35,8 @@ def extract(source_file, unzip_path, &block)
         tables = NdrImport::File::Registry.tables(filename,
                                                   table_mapping.try(:format),
                                                   'unzip_path' => unzip_path,
-                                                  'col_sep' => table_mapping.try(:delimiter))
+                                                  'col_sep'    => table_mapping.try(:delimiter),
+                                                  'root_node'  => table_mapping.try(:root_node))
```

Comment:
Passing through options explicitly like this isn't ideal, and will only get messier as functionality continues to expand.

Not something to block this PR, but could we look to define `all_valid_options` for different `Table` subclasses, and just pass all options though?

It seems out-of-scope, for example, for the `UniversalImporterHelper` to have to know of the `:delimiter` -> `col_sep` mapping - this should only concern `Delimited` ?